src/core/ext/filters/http/server/http_server_filter.cc

```diff
@@ -59,6 +59,15 @@ void FilterOutgoingMetadata(ServerMetadata* md) {
                                        PercentEncodingType::Compatible);
   }
 }
+
+ServerMetadataHandle MalformedRequest(absl::string_view explanation) {
+  auto* arena = GetContext<Arena>();
+  auto hdl = arena->MakePooled<ServerMetadata>(arena);
+  hdl->Set(GrpcStatusMetadata(), GRPC_STATUS_UNKNOWN);
+  hdl->Set(GrpcMessageMetadata(), Slice::FromCopiedString(explanation));
```

Comment:
Couldn't we use `FromStaticString()` in this case?

Reply:
done